### PR TITLE
UI fixes

### PR DESCRIPTION
**Pull request by Ekta29yadav**
Prerak mobilizer text changes
Learner profile- education -detail background color change

### CODE REVIEW

File: apps/front-end/src/component/AgChip.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ export function ChipStatus({ status, ...props }) {
         setNewStatus(t("SHORTLISTED"));
         setColor("blueText.350");
         break;
-      case "prereak_mobilizer":
+      case "prerak_mobilizer":
```
Comment (sagarkoshti1990) [inline apps/front-end/src/component/AgChip.js:42]: remove facilitator status from beneficiary status chip and also change file name to `BeneficiaryStatus`

File: lib/common-lib/src/components/frontend_component/Chip.js [hunk 2]
```diff
@@ -39,7 +39,7 @@ export function ChipStatus({ status, ...props }) {
         setNewStatus(t('SHORTLISTED_FOR_ORIENTATION'))
         setColor('shortlistedColor')
         break
-      case 'prereak_mobilizer':
+      case 'prerak_mobilizer':
```
[the remark this answers is not in the document]
Comment (Ekta29yadav) [reply lib/common-lib/src/components/frontend_component/Chip.js]: done

File: apps/front-end/src/pages/front-end/Dashboard.js [hunk 3]
```diff
@@ -508,7 +508,7 @@ const InfoBox = ({ status }) => {
         </HStack>
       );
       break;
-    case "prereak_mobilizer":
+    case "prerak_mobilizer":
```
[the remark this answers is not in the document]
Comment (Ekta29yadav) [reply apps/front-end/src/pages/front-end/Dashboard.js]: done

File: apps/front-end/src/pages/front-end/BenificiaryListView.js [hunk 4]
```diff
@@ -166,7 +166,7 @@ export default function PrerakListView({ userTokenInfo, footerLinks }) {
       <VStack>
         <Pressable
           onPress={(e) => {
-            ["prereak_mobilizer"].includes(facilitator.status) &&
+            ["prerak_mobilizer"].includes(facilitator.status) &&
```
[the remark this answers is not in the document]
Comment (Ekta29yadav) [reply apps/front-end/src/pages/front-end/BenificiaryListView.js]: done

File: apps/front-end/src/pages/admin/facilitator/view/StatusButton.js [hunk 5]
```diff
@@ -124,9 +124,9 @@ export default function StatusButton({ data, setData }) {
         setDisabledBtn(["screened", "quit", "rusticate"]);
         break;
       case "shortlisted_for_orientation":
-        setDisabledBtn(["prereak_mobilizer", "rejected", "quit", "rusticate"]);
+        setDisabledBtn(["prerak_mobilizer", "rejected", "quit", "rusticate"]);
```
[the remark this answers is not in the document]
Comment (Ekta29yadav) [reply apps/front-end/src/pages/admin/facilitator/view/StatusButton.js]: done

File: apps/front-end/src/pages/admin/facilitator/view/StatusButton.js [hunk 6]
```diff
@@ -55,7 +55,7 @@ const statusList = [
     name: "SHORTLIST_FOR_ORIENTATION",
   },
   {
-    status: "prereak_mobilizer",
+    status: "prerak_mobilizer",
```
[the remark this answers is not in the document]
Comment (Ekta29yadav) [reply apps/front-end/src/pages/admin/facilitator/view/StatusButton.js]: done

File: apps/front-end/src/component/Chip.js [hunk 7]
```diff
@@ -39,7 +39,7 @@ export function ChipStatus({ status, ...props }) {
         setNewStatus(t("SHORTLISTED_FOR_ORIENTATION"));
         setColor("shortlistedColor");
         break;
-      case "prereak_mobilizer":
+      case "prerak_mobilizer":
```
[the remark this answers is not in the document]
Comment (Ekta29yadav) [reply apps/front-end/src/component/Chip.js]: done

File: apps/front-end/src/component/AgChip.js (hunk 1 above)
Comment (Ekta29yadav) [reply apps/front-end/src/component/AgChip.js]: done

**Pull request merged**